dist/usrbg.css

```diff
@@ -35888,4 +35888,5 @@
 .userPopout-3XzG_A [data-user-id="283608915881754625"] {
 	--user-background: url('https://i.imgur.com/ooDH9MH.gif');
 	--user-popout-position: center !important;
-}
\ No newline at end of file
+}
+//https://i.imgur.com/ooDH9MH.gif
```

Comment:
What is this...?

Reply:
oh shoot, my bad pasted in the wrong line...